packages/firebase/compat/index.d.ts

```diff
@@ -6018,6 +6018,25 @@ declare namespace firebase.database {
      * ```
      */
     app: firebase.app.App;
+    /**
+     * INTERNAL methods for internal-use only (tests, etc.).
```

Comment:
Is this accurate? I thought `forceWebSockets` and `forceLongPolling` are for end users. Maybe "additional methods for debugging and special cases"?

Reply:
We need `forceWebSockets` back _for end users_ or we need to abandon Firebase - which has otherwise served us well.

See: https://github.com/firebase/firebase-js-sdk/issues/5433#issuecomment-1124260695